.github/workflows/packaging.yml

```diff
@@ -53,3 +53,11 @@ jobs:
         shell: 'script -q -e -c "bash {0}"'
         run: |
           ./docker/debs/build-and-install-debs.sh deb10 docker/debs/Debian_10/Debian10.dockerfile
+  build-debian11-debs:
```

Comment:
I would love if you could please rename the Debian 10 job to `debian10` to be aligned to the new name.

Reply:
Done.